common.mk

```diff
@@ -0,0 +1,57 @@
+SHELL := bash
+
+TAG       := $(shell date -u +%Y%m%d.%H%M%S)
```

Comment:
This source repository pushes to public Docker Hub repositories.  We may have external users who depend on the existing tagging scheme, and so it was carried across, contrary to internal practice.